app/employee-app/backend/modules/email/types.bal

```diff
@@ -0,0 +1,66 @@
+//
+// Copyright (c) 2005-2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+// 
+import employee_app.types;
+
+# [Configurable] Email configuration record.
+public type EmailAlertConfig record {|
+    # Sender email address
+    string 'from;
+    # Name of the sender
+    string fromName?;
+    # Email subject
+    string subject;
+|};
+
+# [Configurable] Email alerting service configuration record.
+public type EmailAlertOfferConfig record {|
+    # Authorized app UUID provided by the Email service
+    string uuid;
+    # Sender email address
+    string 'from;
+    # ID of the email template
+    string templateId;
+|};
+
+# Payload of the email alerting service.
+public type EmailRecord record {
+    # Application uuid
+    string appUuid = "";
```

Comment:
Is there a specific reason to assign `""` to this field?
I'd suggest using nilable fields and assigning nil as the default value in these cases.

Reply:
This record has removed now.